modules/tinymce/src/plugins/accordion/test/ts/webdriver/AccordionBackspaceDeleteTest.ts

```diff
@@ -105,7 +105,7 @@ describe('webdriver.tinymce.plugins.accordion.AccordionBackspaceDeleteTest', ()
 
     const testPreventBodyDeletion = (deletionKey: DeletionKey) => async () => {
       const editor = hook.editor();
-      editor.setContent(createAccordion(editor, { body: '<p><br></p>' }));
+      createAccordion(editor, { body: '<p><br></p>' });
```

Comment:
Is this correct the old code would do a `editor.setContent(editor.setContent(getAccordionContent(spec))` the new one will be: editor.setContent(getAccordionContent(spec) so skipped one extra setContent filtering. Is that intended, pretty strange test.

Reply:
Yes, this is correct. `editor.setContent` no longer returns a value, so this is no longer possible to do. If you want a similar effect you'd have to do a `getContent` as well, and I wasn't sure we wanted to do that.